cogip/models/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .models import CtrlModeEnum, DynObstacleList, DynObstacleRect, DynRoundObstacle, Vertex, Pose, RobotState  # noqa: F401
```

Comment:
In https://github.com/cogip/simulation/pull/39/commits/cf3734eaf6aab69b8d3ca149f668e3da1a4016bb commit message:
s/This/These/